meta/appveyor.yml

```diff
@@ -41,8 +44,18 @@ build_script:
   - cargo unit-test
   - cargo wasm
   - cargo schema
+  - docker build --pull -t "cosmwasm/cw-gitpod-base:${APPVEYOR_REPO_COMMIT}" .
+  - \[ "${APPVEYOR_REPO_BRANCH}" = "main" \] && image_tag=latest || image_tag=${APPVEYOR_REPO_TAG_NAME}
+  - docker tag "cosmwasm/cw-gitpod-base:${APPVEYOR_REPO_COMMIT}" "cosmwasm/cw-gitpod-base:${image_tag}"
+
+on_success:
+  # publish docker image
+  - docker login --password-stdin -u "$DOCKER_USER" <<<"$DOCKER_PASS"
+  - docker push
+  - docker logout
 
 # whitelist long living branches
 branches:
   only:
-    - master
+    - main
+    - /^v.*/
```

Comment:
Can you use the expression from https://www.appveyor.com/docs/branches/ in order to avoid matches on branches that start with `v` like `very-cool-feature`